en/sim_gazebo_gz/README.md

```diff
@@ -31,16 +31,37 @@ sudo apt-get install gz-garden
 
 ## Running the Simulation
 
-Gazebo SITL simulation can be conveniently run through a `make` command as shown below:
+Gazebo SITL simulation can be conveniently run using a `make` command as shown below:
 
 ```sh
 cd /path/to/PX4-Autopilot
 make px4_sitl gz_x500
 ```
 
-This will run both the PX4 SITL instance and the Gazebo client.
+This runs both the PX4 SITL instance and the Gazebo client.
 Note that all gazebo make targets have the prefix `gz_`.
 
+Another way that Gazebo SITL can be connected is in standalone mode.
+This means that PX4 and Gazebo will be started in separate terminals.
+You can start PX4 with:
+
+```sh
+cd /path/to/PX4-Autopilot
+GZ_PX4_STANDALONE=1 make px4_sitl gz_x500
```

Comment:
Why do we need to specify `gz_x500` if the make command is not going to launch it?

Reply:
The `make` command will launch PX4, it is just waiting in a holding loop, trying to connect to an instance of Gazebo Sim. Once an instance of gz-server, then PX4 will connect as normal.